ompi/errhandler/errcode.c

```diff
@@ -34,84 +36,84 @@
 #include "ompi/constants.h"
 
 /* Table holding all error codes */
-opal_pointer_array_t ompi_mpi_errcodes;
+opal_pointer_array_t ompi_mpi_errcodes = {{0}};
 int ompi_mpi_errcode_lastused=0;
 int ompi_mpi_errcode_lastpredefined=0;
 
-ompi_mpi_errcode_t ompi_success;
-ompi_mpi_errcode_t ompi_err_buffer;
-ompi_mpi_errcode_t ompi_err_count;
-ompi_mpi_errcode_t ompi_err_type;
-ompi_mpi_errcode_t ompi_err_tag;
-ompi_mpi_errcode_t ompi_err_comm;
-ompi_mpi_errcode_t ompi_err_rank;
-ompi_mpi_errcode_t ompi_err_request;
-ompi_mpi_errcode_t ompi_err_root;
-ompi_mpi_errcode_t ompi_err_group;
-ompi_mpi_errcode_t ompi_err_op;
-ompi_mpi_errcode_t ompi_err_topology;
-ompi_mpi_errcode_t ompi_err_dims;
-ompi_mpi_errcode_t ompi_err_arg;
-ompi_mpi_errcode_t ompi_err_unknown;
-ompi_mpi_errcode_t ompi_err_truncate;
-ompi_mpi_errcode_t ompi_err_other;
-ompi_mpi_errcode_t ompi_err_intern;
-ompi_mpi_errcode_t ompi_err_in_status;
-ompi_mpi_errcode_t ompi_err_pending;
+ompi_mpi_errcode_t ompi_success = {{0}};
+ompi_mpi_errcode_t ompi_err_buffer = {{0}};
```

Comment:
Yeah, really weird. If both sets are static it will be only slightly less weird.